task.yaml

```diff
@@ -6,7 +6,7 @@ task:
     # Add ability to pull container back
     commit_on_stop: true
     # image name to start on miner, required param
-    name: image:v1
+    name: httpd:latest
```

Comment:
?

Reply:
You have to change image name to something meaningful every time you test tasks. Somebody had to put an end to this. I think @sshaman1101 is with me here.